test/net/sourceforge/kolmafia/request/ClanLoungeRequestTest.java

```diff
@@ -0,0 +1,53 @@
+package net.sourceforge.kolmafia.request;
+
+import static internal.helpers.Player.setupFakeResponse;
+import static internal.helpers.Preference.isSetTo;
+import static org.hamcrest.MatcherAssert.assertThat;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import net.sourceforge.kolmafia.KoLCharacter;
+import net.sourceforge.kolmafia.preferences.Preferences;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+public class ClanLoungeRequestTest {
+  @BeforeEach
+  public void init() {
+    Preferences.saveSettingsToFile = false;
+    KoLCharacter.reset("ClanLoungeRequestTest");
+    Preferences.reset("ClanLoungeRequestTest");
+    // don't try to visit the fireworks shop
+    Preferences.setBoolean("_fireworksShop", true);
+  }
+
+  static String html(String path) {
```

Comment:
This is a nice method. It could be used all over the place. I have added a version which does the same but doesn't catch the IOException in many test files.

Perhaps you could put it somewhere in internal/helpers. Perhaps even in Player, although that is almost exclusively Cleanups. Almost; somebody but two variants for countItem there, as wrappers for InventoryManager.getCount.

Reply:
This reminds me that a while back I added some extra request / response asserts to `AbstractCommandTestBase`, but only used them in one class. These too could do with being used elsewhere.